app/src/main/java/com/github/houseorganizer/houseorganizer/MainScreenActivity.java

```diff
@@ -78,27 +78,23 @@ private void rotateView(View v) {
 
     private void addEvent(View v) {
         Map<String, Object> data = new HashMap<>();
-        Event event = new Event("added", "this is the event that i added using the add button", LocalDateTime.now(), 100);
         data.put("title", "added");
         data.put("description", "this is the event that i added using the add button");
-        data.put("start", LocalDateTime.now().toEpochSecond(ZoneOffset.UTC));
+        data.put("start", LocalDateTime.now().plusHours(2).toEpochSecond(ZoneOffset.UTC));
```

Comment:
This is for testing purpose is that it ? Just to be sure : in the future the starting time will be decided by the user ?

Reply:
yes thats what im implementing in the feature/event_editing branch this sprint

Reply:
We might also need to consider various timezones when adding events, our users will not all live in UTC

Reply:
I dont think it is a problem because all we do is store it using UTC and then load it back from UTC so the part where it uses UTC is the backend from how i understand it.